### DISCUSSION TITLE
Image magick on azure

### DISCUSSION THREAD

**Discussion by ivanamar996** [ImageMagick/ImageMagick#6334] [Help]
### ImageMagick version

7.1.0-55

### Operating system, version and so on

Windows

### Description

Hi,

Is there any other distribution of image magick that does not require access to the registry?

We are trying run image magick commands from c# code by starting new process with magick.exe, but it is failing when the code is deployed on azure app service with error: 

magick.exe: RegistryKeyLookupFailed `CoderModulesPath' @ error/module.c/GetMagickModulePath/667.
magick.exe: no decode delegate for this image format `PNG' @ error/constitute.c/ReadImage/741.

Kind regards,
Ivana

**Comment by snibgo** [comment]
See https://imagemagick.org/script/download.php#windows and search for "portable", for example:
```
ImageMagick-7.1.1-9-portable-Q16-x64.zip 	Portable Win64 static at 16 bits-per-pixel component. Just copy to your host and run (no installer, no Windows registry entries).
```
